assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -1,7 +1,7 @@
 import * as d3 from 'd3'
```

Comment:
This change was made due since the event is passed as one of the arguments and cannot be accessed as `d3.event`

More on this https://observablehq.com/@d3/d3v6-migration-guide#events

Reply:
Lots of warnings were thrown during the `docker compose up ` w.r.t to `d3.event`. So fixes were based on those alerts and the migration guide help